<launch>
	<!-- Joystick device path.
	Default is /dev/input/rumblepad. If an environment variable JOYSTICK_DEVICE
	is set then this replaces the default. If the launch file is started with
	the argument js:=blah then blah overides both the default and the environment. -->
	<arg name="js" default="$(optenv JOYSTICK_DEVICE /dev/input/rumblepad)" />

	<arg name="machine" default="localhost" />
	<arg name="user" default="" />


	<!-- now has an IF checking for the arg "remote" -->
	<machine name="$(arg machine)" address="$(arg machine)" env-loader="$(optenv ROS_ENV_LOADER )" user="$(arg user)" default="true" />


	<!-- Joystick driver node -->
	<node pkg="joy" type="joy_node" name="joystick">
		<param name="dev" value="$(arg js)" type="string"/>
		<param name="autorepeat_rate" value="10.0" type="double"/>
	</node>

	<!-- Rumblepad control -->
	<node pkg="scitos_teleop" type="rumble_control" name="teleop_joystick" output="screen" />

</launch> 
